docs/web/usecases-implement-component-participate-app-config.mdx

```diff
@@ -1,83 +1,40 @@
 
 Geocortex Web has a powerful [app configuration](concepts-app-config.mdx) model which can be used to easily change the behavior of an application without modifying custom code. Using app config to power a components behavior increases its reusability and customizability.
 
 By the end of this article, you'll have the knowledge to build a component that displays relevant news items at the top of your map. These news items will be populated from config, along with a value that tells the news component whether or not to be visible by default.
 
 ## Prerequisites
 
-Learn how to [implement a custom component](usecases-implement-component-participate-app-config.mdx).
+<PrereqsCustomCodeSnippet />
 
-## Component Models
+## Starting Point
 
-Each component has an associated model. The model is the backing data source for a component, and usually has properties that are initialized from configuration. It is best practice to treat the model for a component as the state, and single source of truth, for data values that the component consumes.
-
-By configuring built-in [layout elements](concepts-layout.mdx) such as the `<map>`, you've already used models. In this example, the `<map>` component is linked to the `map-extension` model in the configuration.
-
-<Tabs
-  defaultValue="layout"
-  values={[
-    { label: "Layout", value: "layout" },
-    { label: "App Config", value: "config" },
-  ]}>
-<TabItem value="layout">
-
-```xml
-<?xml version="1.0" encoding="utf-8" ?>
-<layout xmlns="https://geocortex.com/layout/v1">
-    <map config="my-map-config" />
-</layout>
-```
-
-</TabItem>
-<TabItem value="config">
-
-```json
-{
-    "schemaVersion": "1.0",
-    "items": [
-        {
-            "$type": "map-extension",
-            "id": "my-map-config",
-            "onInitialized": [
-                {
-                    "name": "ui.alert",
-                    "arguments": {
-                        "message": "Hello World"
-                    }
-                }
-            ]
-        }
-    ]
-}
-```
-
-</TabItem>
-</Tabs>
-
-## Example: Model for the News Feed Component
-
-We are going to add configuration to the [news feed component](usecases-implement-component-with-ui.mdx#example-rss-feed-of-news-updates) we created earlier. The `hidden` state property and `newsItems` property on the existing component can be moved into configuration consumed by the component model.
-
-### Starting Point
-
-We will start this example from the News Feed component and model built in the [previous example](usecases-implement-component-with-ui.mdx#example-rss-feed-of-news-updates).
+We are going to add configuration to a custom component that displays news items. This component currently is following bad practices and does not treat the model as the source of truth for it's data. We are going to move the `newsItems` list to a configurable property on the model, and add a new configurable property, `hideOnStartup`.
 
 <FrameworkComponentsSnippet />
```

Comment:
Let's remove any reference to "framework" i think. This could instead read something like:

> This example uses Geocortex Web UI Components

However there is some ambiguity between layout elements which we call components, and our exported react cmomponents for use within the layout element ... components :stuck_out_tongue: 

Would be good to figure out how we want to talk about these currencies as it's a little confusing right now IMO. 

cc @eric-parsons

Reply:
Replaced all references to "Geocortex Framework UI Components" with "Geocortex Web UI Components"

Reply:
Just throwing some ideas for how to talk about the currrently without mentioning the framework

"Geocortex Themed React Components"
"Geocortex React Component Library"
Geocortex Styled React Components"

Reply:
I think in the past we threw around the idea that when it's necessary to be specific, use "layout component" when referring to the component that corresponds to a layout element, and "React component" to talk about React components in general. But really the former is just a special case of the latter -- a layout component is a React component in every sense, just not the other way around. The only thing that makes layout components a bit different from normal React patterns is that a layout component should never directly render other layout components -- the framework will create this hierarchy indirectly via children. Not sure if that helps?

Reply:
Using "Layout Component" when we need to be more specific makes sense to me.

Part of me wants to be more specific and specify "Geocortex Layout Components" just to make the fact that this are special components provided by the framework stand out.

Reply:
@eric-parsons @ianschmitz what do you think of "Geocortex Layout Components?" Or "Geocortex Web Layout Components"?

Reply:
Of those two, I prefer "Geocortex Web layout components".

Reply:
Changed all references to Geocortex Web layout components.